types/gtag.js/index.d.ts

```diff
@@ -7,10 +7,16 @@ declare var gtag: Gtag.Gtag;
 declare namespace Gtag {
   interface Gtag {
     (command: 'config', targetId: string, config?: ControlParams | EventParams | CustomParams): void;
-    (command: 'set', targetId: string, config: CustomParams): void;
+    (command: 'set', targetId: string, config: CustomParams | boolean): void;
```

Comment:
Fixed the following points @rdalmas made : https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52148#issuecomment-819631093

> I've tried to specify for example a developer Id using gtag and I'm not able. Example would be this:
> gtag('set', 'developer_id', true)